src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/OrderItem.sql

```diff
@@ -0,0 +1,28 @@
+﻿CREATE TABLE dbo.OrderItem
+(
+    OrderItemId int IDENTITY(1,1) NOT NULL,

```

Comment:
Space after comma, e.g. `IDENTITY(1, 1)`? I find that slightly more readable but it's such a small thing...